spyder/plugins/variableexplorer/widgets/collectionsdelegate.py

```diff
@@ -119,7 +119,7 @@ def createEditor(self, parent, option, index, object_explorer=False):
                 message = _("Spyder is unable to show the {val_type} or object"
                             " you're trying to view because <tt>{module}</tt>"
                             " is not installed. ")
-                if running_in_mac_app():
+                if is_conda_based_app():
                     message += _("Please consider using the full version of "
                                  "the Spyder MacOS application.<br>")
```

Comment:
I think this could be removed because we're not planning to release light versions of our conda installers, right?

Reply:
Yeah, I suppose we should discuss that. At the very least, `MacOS` should be removed from the message.